poetry.lock

```diff
@@ -2530,15 +2442,14 @@ tests = ["pytest", "simplejson"]
 
 [[package]]
 name = "mcp"
-version = "1.13.1"
+version = "1.14.1"
```

Comment:
Noted here: also bumping `mcp` from `1.31.1` to `1.14.1`. (Issue was present in both versions.)